### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by vinkla**
Added `ruleset.xml` and `.sensiolabs.yml`.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 /.gitignore export-ignore
 /.scrutinizer.yml export-ignore
+/.sensiolabs.yml export-ignore
 /.styleci.yml export-ignore
 /.travis.yml export-ignore
 /CHANGELOG.md export-ignore
 /CONTRIBUTING.md export-ignore
 /phpunit.xml.dist export-ignore
 /README.md export-ignore
+/ruleset.xml
```
Comment (gabrielbull) [inline .gitattributes]: Should `/ruleset.xml` also have the tag `export-ignore`?

Comment (vinkla) [inline .gitattributes]: Fixed.

**Comment by vinkla** [comment]
:+1:

**Comment by gabrielbull** [comment]
:snowflake::sparkles::snowflake:
:sparkles::star2::sparkles:
:snowflake::sparkles::snowflake:

**Comment by vinkla** [comment]
sparkle

**Pull request merged**